pkg/cloudevents/transport/nats/codec_test.go

```diff
@@ -378,13 +367,6 @@ func TestCodecAsMiddleware(t *testing.T) {
 					got.Data = data
 				}
 
-				if tc.wantErr != nil || err != nil {
```

Comment:
the tests need to be expanded to non-happy path only. Please leave this?

Reply:
ok